src/services/collaborativesearch.service.ts

```diff
@@ -774,82 +828,42 @@ export class CollaborativesearchService {
+                        , notgwithinForGet, notgintersectForGet, dateformat, false, this.max_age, this.fetchOptions)
                 );
                 break;
             case projType.search.valueOf():
-                search = <Search>projection[1];
-                includes = [];
-                excludes = [];
-                if (search.projection !== undefined) {
-                    if (search.projection.excludes !== undefined) {
-                        excludes.push(search.projection.excludes);
-                    } if (search.projection.includes !== undefined) {
-                        includes.push(search.projection.includes);
-                    }
-                }
-                let sort: string;
-                if (search.sort === undefined) {
-                    sort = null;
-                } else {
-                    if (search.sort.sort === undefined) {
-                        sort = null;
-                    } else {
-                        sort = search.sort.sort;
-                    }
-                }
                 result = <Observable<Hits>>from(
                     this.exploreApi.search(this.collection, fForGet, qForGet
                         , pwithinForGet, gwithinForGet, gintersectForGet, notpwithinForGet
-                        , notgwithinForGet, notgintersectForGet, false, includes, excludes, search.size.size,
-                        search.size.from, sort, this.max_age, this.fetchOptions)
+                        , notgwithinForGet, notgintersectForGet, dateformat, false, flat, includes, excludes, pageSize,
+                        pageFrom, pageSort, pageAfter, this.max_age, this.fetchOptions)
                 );
                 break;
             case projType.geosearch.valueOf():
-                search = <Search>projection[1];
-                includes = [];
-                excludes = [];
-                if (search.projection !== undefined) {
-                    if (search.projection.excludes !== undefined) {
-                        excludes.push(search.projection.excludes);
-                    } if (search.projection.includes !== undefined) {
-                        includes.push(search.projection.includes);
-                    }
-                }
                 result = <Observable<FeatureCollection>>from(
                     this.exploreApi.geosearch(this.collection, fForGet, qForGet
                         , pwithinForGet, gwithinForGet, gintersectForGet, notpwithinForGet
-                        , notgwithinForGet, notgintersectForGet, false, isFlat, includes, excludes, search.size.size,
-                        search.size.from, null, this.max_age, this.fetchOptions)
+                        , notgwithinForGet, notgintersectForGet, dateformat, false, flat, includes, excludes, pageSize,
+                        pageFrom, pageSort, pageFrom, this.max_age, this.fetchOptions)
```

Comment:
is it pageAfter instead of pageFrom (the param just before this.max_age) ?

Reply:
yes